docs/cloud/dev/spryker-cloud-commerce-os/performance-testing-in-staging-enivronments.md

```diff
@@ -14,4 +15,592 @@ If you want to execute a full load test on a production-like dataset and traffic
+git clone https://github.com/spryker-shop/b2c-demo-shop.git ./
+git clone git@github.com:spryker/docker-sdk.git docker
+```
+
+### Integrating Gatling
+
+With the B2C Demo Shop and Docker SDK cloned, you will need to make a few changes to integrate Gatling into your project. These changes include requiring the load testing tool with composer as well as updating the [Router module](/docs/scos/dev/migration-concepts/silex-replacement/router/router-yves.html) inside of Yves. 
+
+{% info_block infoBox %}
+
+The required composer package as well as the changes to the Router module are needed on the project level. They are what help to run the appropriate tests and generate the data needed for the load test tool.
+
+It should be noted that the Spryker Suite already has these changes implemented in them and comes pre-configured for load testing. For either of the B2C or B2B Demo Shops, you will need to implement the below changes.
+
+{% endinfo_block %}
+
+1. Require the *composer* package. This step is necessary if you are looking to implement the Gatling load testing tool into your project. This line will add the new package to your `composer.json` file. The `--dev` flag will install the requirements needed for development which have a version constraint (e.g. "spryker-sdk/load-testing": "^0.1.0").
+
+```bash
+composer require spryker-sdk/load-testing --dev
+```
+
+2. Add the Router provider plugin to `src/Pyz/Yves/Router/RouterDependencyProvider.php`. Please note that you must import the appropriate class, `LoadTestingRouterProviderPlugin` to initialize the load testing. We also need to build onto the available array, so the `return` clause should be updated to reflect the additions to the array with `$routeProviders`.
+
+```php
+<?php
+
+...
+
+namespace Pyz\Yves\Router;
+
+...
+use SprykerSdk\Yves\LoadTesting\Plugin\Router\LoadTestingRouterProviderPlugin;
+...
+
+class RouterDependencyProvider extends SprykerRouterDependencyProvider
+{
+    ...
+    
+    /**
+     * @return \Spryker\Yves\RouterExtension\Dependency\Plugin\RouteProviderPluginInterface[]
+     */
+    protected function getRouteProvider(): array
+    {
+        $routeProviders = [
+        ...
+        ];
+		
+        if (class_exists(LoadTestingRouterProviderPlugin::class)) {
+            $routeProviders[] = new LoadTestingRouterProviderPlugin();
+        }
+        
+        return $routeProviders;
+    }
+
+    ...
+}
+```
+
+3. Make sure the `fixtures` command is enabled in the main `codeception.yml`:
+
+```yaml
+...
+extensions:
+  commands:
+    - \SprykerTest\Shared\Testify\Fixtures\FixturesCommand
+...
+```
+
+### Setting up the environment
+
+{% info_block infoBox %}
+
+This step is only needed for a new project, such as our B2C Demo Shop example. Otherwise, if you have a pre-existing project, once the above changes for integrating Gatling have been made, you merely need to re-built the application (such as with `docker/sdk up --build --assets --data`) to apply the changes.
```

Comment:
```suggestion
This step is only needed for a new project, such as our B2C Demo Shop example. Otherwise, if you have a pre-existing project, once the above changes for integrating Gatling have been made, you merely need to re-build the application (such as with `docker/sdk up --build --assets --data`) to apply the changes.
```